src/OceanSeaIceModels/time_step_ocean_sea_ice_model.jl

```diff
@@ -41,7 +41,7 @@ function time_step!(coupled_model::OceanSeaIceModel, Δt; callbacks=[], compute_
     time_step!(ocean)
 
     # Time step the atmosphere
-    time_step!(atmosphere, Δt)
+    time_step!(atmosphere)
```

Comment:
```suggestion
    time_step!(atmosphere, Δt)
```
we probably need this if atmospheres need to adapt their `Δt` to the coupled simulation's `Δt`

Reply:
No, I don't want to use a different design for different simulation types. I am going to design a separate interface for setting dt.

Reply:
I have rethought this and I think it's a good idea to combine these